synapse/lib/storm.py

```diff
@@ -388,3 +394,167 @@ def runStormCmd(self, snap, genr):
             node = snap.getNodeByBuid(buid)
             if node:
                 yield node, node.initPath()
+
+class RefsCmd(Cmd):
+    '''
+    Get nodes adjacent to inbound nodes.
+    '''
+    name = 'refs'
+    def getArgParser(self):
+        pars = Cmd.getArgParser(self)
+        pars.add_argument('-d', '--degrees', type=int, default=1, action='store',
```

Comment:
Consider adding choices here to constrain silliness and you can remove the check in `runStormCmd`.  `choices=xrange(1, 10)` or some such

Reply:
per discussion, we're allowing a user to have as much silliness as possible.

Reply:
:rofl: